saml/src/main/java/com/linecorp/armeria/server/saml/SamlAssertionConsumerConfigBuilder.java

```diff
@@ -34,9 +35,18 @@
         this.parent = parent;
```

Comment:
Could delegate to `this()` with 2 args.

Reply:
In order to do that I need to add `@Nullable` to `SamlEndpoint` parameter which I didn't want it. If you strongly recommend it, I can change that. 😄